dl-win32.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 
-GTK_VERSION=4.8.1-1
+GTK_VERSION=4.8.2-1
```

Comment:
Can't this be automated somehow as well?

Reply:
working on it